products/jse/features/jse_landing_page.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Ensure Site Owners can create and delete landing pages
+  In order to ensure that a site owner can create, edit and delete landing pages
+  As a site owner
+  I want to be able create, edit and delete landing pages
+
+  Background:
+  Given I am logged in as a user with the "site owner" role
+
+  @api @dev @destructive @javascript
+  Scenario: Ensure site owners can create, edit and delete their own landing pages
```

Comment:
Site owners should be able to edit any landing page. 
Consider having the current site owner log out and a new site owner edit and delete the page.